File: app/src/test/java/org/worshipsongs/registry/DummyFragment1.java
package org.worshipsongs.registry;

import android.os.Bundle;


import org.worshipsongs.listener.SongContentViewListener;

import androidx.fragment.app.Fragment;

/**
 * @author  Madasamy
 * @since 3.x.x
 */

public class DummyFragment1 extends Fragment implements ITabFragment
{
    @Override
    public int defaultSortOrder()
    {
        return 1;
    }

    @Override
    public String getTitle()
    {
        return "title";
    }

    @Override
    public boolean checked()
    {
        return true;
    }

    @Override
    public void setListenerAndBundle(SongContentViewListener songContentViewListener, Bundle bundle)
    {

    }
}

File: app/src/test/kotlin/org/worshipsongs/registry/DummyFragment1.kt
package org.worshipsongs.registry

import android.os.Bundle
import androidx.fragment.app.Fragment
import org.worshipsongs.listener.SongContentViewListener

/**
 * @author  Madasamy
 * @since 3.x.x
 */
class DummyFragment1 : Fragment(), ITabFragment {
    override fun defaultSortOrder(): Int {
        return 1
    }

    override val title: String
        get() = "title"

    override fun checked(): Boolean {
        return true
    }

    override fun setListenerAndBundle(
        songContentViewListener: SongContentViewListener?,
        bundle: Bundle
    ) {
    }
}